### PR TITLE
system: update licenses to Apache

### PR DESCRIPTION
**Pull request by jerpelea**
## Summary

Gregory Nutt is has submitted the SGA

Ken Pettit has submitted the ICLA

as a result we can migrate the licenses to Apache.

## Impact
LICENSE

## Testing
NONE

### CODE REVIEW

**Review by xiaoxiang781216 — APPROVED** [review]
LGTM.

**Comment by jerpelea** [comment]
please ignore the remaining nxstyle errors

File: system/ping6/ping6.c [hunk 1]
```diff
@@ -204,7 +198,8 @@ static void ping6_result(FAR const struct ping6_result_s *result)
                    (result->nrequests >> 1)) /
                    result->nrequests;
 
-            printf("%u packets transmitted, %u received, %u%% packet loss, time %d ms\n",
+            printf("%u packets transmitted, %u received,
+                    %u%% packet loss, time %d ms\n",
```
Comment (Ouss4) [inline system/ping6/ping6.c:202]:
```suggestion
            printf("%u packets transmitted, %u received, "
                     "%u%% packet loss, time %d ms\n",
```

**Comment by jerpelea** [comment]
@xiaoxiang781216 please ignore the remaining nxstyle errors

**Review by xiaoxiang781216 — APPROVED** [review]

**Pull request merged**